docs/openapi/components/schemas/common/PostalAddress.yml

```diff
@@ -49,8 +49,7 @@ properties:
   addressCountry:
     title: Address Country
     description: >-
-      Code specifying the country (and country subdivision) for the address
-      using ISO 3166-1.
+      The two-letter ISO 3166-1 alpha-2 country code.
```

Comment:
the schema.org entry for this has this text:
The country. For example, USA. You can also provide the two-letter ISO 3166-1 alpha-2 country code.
is there a reason why we are overloading it?

Reply:
Because we want Traceability to be more definite than schema.org

Reply:
The change here seems super ripe for confusion, as all we're doing is modifying the description, while still linking to the term as defined by addressCountry.
In principle im absolutely all for being more definite but this appears ripe for confusion.

Reply:
I suggest that for our definition, we indicate the ISO code is preferred, but other designations (such as "USA" or "United States") are acceptable.  I suggest that this modified definition also be submitted as a PR against schema.org (upon merge of which, we can drop our distinct definition/property, and just use `schema:addressCountry`).

Reply:
I don't see what is confusing. IMHO it's fine to reuse external props but tighten them to what is more appropriate in your domain
This is called Application Profiles. It can be formalized in JSON Schema, SHACL or SHEX.

I disagree that Traceability should be more permissive since its domain (trade data exchange) is more tightly controlled than schema.org (eating data from billions of sites).
Eg if you don't use the ISO code, you cannot link to UNCEFACT top-level LOCODEs.

Reply:
I completely agree with @VladimirAlexiev on this and find it difficult to actively "go backwards" and loosen up this definition. 

`The two-letter ISO 3166-1 alpha-2 country code.` is about as clear and concise as it can get, and I disagree that it is "super ripe for confusion". 

Let's lead (schema.org) by example here.

Reply:
I'm still not a fan, but I won't block this.